protobuf/src-proto/Proto/Proto/Riak.hs

```diff
@@ -0,0 +1,11938 @@
+{- This file was auto-generated from proto/riak.proto by the proto-lens-protoc program. -}
+{-# LANGUAGE ScopedTypeVariables, DataKinds, TypeFamilies,
+  UndecidableInstances, GeneralizedNewtypeDeriving,
+  MultiParamTypeClasses, FlexibleContexts, FlexibleInstances,
+  PatternSynonyms, MagicHash, NoImplicitPrelude, DataKinds #-}
+{-# OPTIONS_GHC -fno-warn-unused-imports#-}
+{-# OPTIONS_GHC -fno-warn-duplicate-exports#-}
```

Comment:
If these `OPTIONS_GHC` are included here we can drop them from the `ghc-options:` section of the cabal file.